test/functionalTest/cases/3176_entity_without_attributes_not_listed/entity_without_attributes_not_listed.test

```diff
@@ -0,0 +1,126 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+POST Entities V2 No Attributes
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0
+
+--SHELL--
+#
+# 01. POST /v2/registrations
+# 02. POST /v2/entities
+# 03. GET /v2/entities
+#
+
+
+echo "01. POST /v2/registrations"
+echo "=========================="
+payload='{
+  "id": "5aeb13827d4ef10a12a02630",
+  "dataProvided": {
+    "entities": [
+      {
+        "id": "urn:ngsi-ld:Vehicle:A456",
```

Comment:
This registration doesn't match with the one being created after (E1 / T1)

Reply:
Although looking to the query (general `GET /v2/entities`) maybe this is intentional...

Reply:
Actually, I did change this "id" for "E1".
When I did, the error didn't manifest, so I changed it back again.

NTC